schema_queries_test.go

```diff
@@ -16,5 +16,9 @@ func TestSchemaQueries(t *testing.T) {
 	session := createSessionFromCluster(cluster, t)
 	defer session.Close()
 
-	assertTrue(t, "keyspace present in schemaDescriber", session.schemaDescriber.cache["gocql_test"].Name == "gocql_test")
+	keyspaceMetadata, ok := session.schemaDescriber.metadata.keyspaceMetadata.getKeyspace("gocql_test")
```

Comment:
You should not access `schemaDescriber` fields directly, please make a wrapper for them that starts with Uppercase letter to show that it is meant to be used instead of direct access, eventually we will evacuate it to a separate package.

Reply:
actually we have `getSchema` in `schemaDescriber` exactly for that, I am not sure why I didn't use it, fixed